examples/chart/teleport-kube-agent/templates/statefulset.yaml

```diff
@@ -0,0 +1,140 @@
+{{- if .Values.storage.enabled }}
+{{- if .Values.teleportVersionOverride }}
+  {{- $_ := set . "teleportVersion" .Values.teleportVersionOverride }}
+{{- else }}
+  {{- $_ := set . "teleportVersion" .Chart.AppVersion }}
```

Comment:
```suggestion
  {{- $_ := set . "teleportVersion" .Chart.Version }}
```

Reply:
Address in 1fa70acd5e5bc3ad7a2422a235fe9ccaff8a9657